src/main/java/com/ericsson/gerrit/plugins/eiffel/handlers/Table.java

```diff
@@ -0,0 +1,32 @@
+/*
+   Copyright 2019 Ericsson AB.
+   For a full list of individual contributors, please see the commit history.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package com.ericsson.gerrit.plugins.eiffel.handlers;
+
+public enum Table {
+    SCS_TABLE("branch"), SCC_TABLE("changeId");
+
+    final String keyName;
+
+    Table(String keyValue) {
+        this.keyName = keyValue;
```

Comment:
What if keyName was public final? then one could access it from my_table.keyName and not getKeyName? would that look better.

Reply:
I'm sorry i couldnt take a look at it today and i won't be here tomorrow. I will ask @saif-ericsson to look into this :)

Reply:
I am leaving it as it is... as it doesn't make any difference.